src/index.ts

```diff
@@ -104,6 +105,7 @@ export const initialize = ({
   collectSessionId,
   dropSingleUserInteractionTraces,
   collectErrors = true,
+  userInteractionElementNameLimit,
```

Comment:
assign the default value here if undefined

Reply:
Ok, done 👍